qcodes/instrument_drivers/Keysight/Infiniium.py

```diff
@@ -72,6 +72,11 @@ def prepare_curvedata(self) -> None:
         assert isinstance(root_instrument, Infiniium)
         root_instrument.trace_ready = True
 
+        # set up the instrument
+        # ---------------------------------------------------------------------
+        # realtime mode: only one trigger is used
+        instr._parent.acquire_mode(acquire_mode)
```

Comment:
Cant you use `root_instrument` as above ?

Reply:
Yes, updating now